sql/mob_skill_lists.sql

```diff
@@ -2692,3 +2692,770 @@ INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2145); -- Gospel of
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2146); -- Void of Repentance "Repent thy irreverence..."
 INSERT INTO `mob_skill_lists` VALUES ('Alexander_LowHP',785,2147); -- Divine Judgement   "Accept thy destruction..."
 
+-- Alkonost
+INSERT INTO `mob_skill_lists` VALUES ('Alkonost',786,576); -- back heel
```

Comment:
Hey, see how the mobname is already a field in the sql line itself? I've pointed this out to you like 50 times now. Cut it out.